single_kernel_mongo/config/relations.py

```diff
@@ -42,3 +42,4 @@ class ExternalProviderRelations(str, Enum):
     """The relations we provide to non mongo related charms."""
 
     COS_AGENT = "cos-agent"
+    LOGGING = "logging"
```

Comment:
note: 
`cos-agent` is only used by VM
`logging`(loki_push_api)  relation is only used by K8s
`grafana_dashboard` is a cos relation that is only used by K8s. I think its missing here 
`prometheus_scrape` is a cos relation that is only used by K8s. I think its missing here

Reply:
if the constant `COS_AGENT ` is not used it can be removed
if the constant `LOGGING` is used, it might be nice to add a comment # note this relation is only used for K8s charms

Reply:
This is now done